### PR TITLE
Fixed sdk import failure on setting up page

### PR DESCRIPTION
**Pull request by siddarthpai**
image
Fixed SDK load error

### CODE REVIEW

File: pages/api/events.ts [hunk 1]
```diff
@@ -1,37 +1,79 @@
...
+    };
+}
+
+type RudderStackEventPropertiesTrack = {
+    userId: string, 
+    anonymousId: string, 
+    event: string, 
+    properties: apiObject;
+    timestamp: Date;
+  };
```
Comment (avikalpg) [inline pages/api/events.ts:22]: Again, I think you should use the types provided by the RudderStack SDK.

**Review by avikalpg — CHANGES_REQUESTED** [review]
Returning null might lead to other issues wherever the returned object is used.

File: utils/rudderstack_initialize.ts [hunk 2]
```diff
@@ -1,90 +1,77 @@
...
-  page: (
-    type: string,
-    pageName: string,
-    properties: { [key: string]: any }
-  ) => void;
+  ) => void
+  page: (type: string, pageName: string, properties: { [key: string]: any }) => void
 }
 
-let rudderAnalyticsInstance: typeof import('rudder-sdk-js') | null = null;
```
Comment (avikalpg) [inline utils/rudderstack_initialize.ts:21]: @MuskanPaliwal This `rudderAnalyticsInstance` is being removed in this PR. Is this fine?

**Review by avikalpg — APPROVED** [review]

File: utils/rudderstack_initialize.ts (hunk 2 above)
Comment (MuskanPaliwal) [reply utils/rudderstack_initialize.ts:21]: Haan it should be. But I this was needed because of the `null` type

**Review by MuskanPaliwal — APPROVED** [review]

**Review by avikalpg — APPROVED** [review]

**Pull request merged**